pkg/actions/import.go

```diff
@@ -271,9 +271,6 @@ func (i *Import) createComponentFromData(name, data string, templateType prototy
 
 func (i *Import) createComponent(fileName, base, ext string, templateType prototype.TemplateType) error {
 	var name bytes.Buffer
-	if i.module != "" {
```

Comment:
does this hold if someone specifies the "/" module explicitly (is that even allowed in this case?)

Reply:
Yup, this should hold for cases with and without `--module /` passed from CLI. Default i.module is `/` if module is not specified explicitly.

Rebased with updated test cases